packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -66,6 +103,29 @@ export class VaTable {
     });
   }
 
+  componentDidUpdate() {
+    // Update the sort icon
+    if (this.sortColumn >= 0) {
+      const icon = this.sortAscending ? descendingIcon : ascendingIcon;
+      const button = this.el.querySelector('va-table-row[slot] button');
+      const node = button.children[0];
+      node.outerHTML = icon;
+    }
+  }
```

Comment:
The `sortAscending` state variable gets updated on sort, so this function will run after a sort.